code/modules/research/techweb/nodes/mecha_nodes.dm

```diff
@@ -57,15 +57,15 @@
 	design_ids = list("gygax_chassis", "gygax_torso", "gygax_head", "gygax_left_arm", "gygax_right_arm", "gygax_left_leg", "gygax_right_leg", "gygax_main",
 	"gygax_peri", "gygax_targ", "gygax_armor")
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 3000)
-*/
+
```

Comment:
This directlys adds it back to the main techweb.

Reply:
i didnt realize i removed that but you could have waited me to fix that instead of immidnetly closing it what the fuck